traits/ctraits.c

```diff
@@ -1538,26 +1540,25 @@ PyDoc_STRVAR(
 PyDoc_STRVAR(
     has_traits_traits_inited_doc,
     "traits_inited()\n"
-    "traits_inited(value)\n"
-    "\n"
-    "Get or set the initialization state of this object.\n"
-    "\n"
-    "With zero arguments, get the state: return True if the object has\n"
-    "already been initialized, and False otherwise.\n"
-    "\n"
-    "With a single argument, if that argument is true, mark the object\n"
-    "as initialized. If the argument is false, the state is unchanged.\n"
     "\n"
-    "Parameters\n"
-    "----------\n"
-    "value : bool\n"
-    "    Only the value True (or another truthy value) should be passed.\n"
+    "Get the initialization state of this object.\n"
     "\n"
     "Returns\n"
     "-------\n"
     "initialized : bool\n"
     "    True if the object is initialized, else False.\n");
 
+PyDoc_STRVAR(
+    has_traits__trait_set_inited_doc,
+    "_trait_set_inited()\n"
+    "\n"
+    "Indicate whether this object has been initialized.\n"
```

Comment:
This sounds a bit as though the method returns something. How about "Declare that this object ..." instead of "Indicate whether this object ..."?